.github/workflows/roll.yaml

```diff
@@ -0,0 +1,21 @@
+name: Roll WebRTC revision
```

Comment:
I assume you mean "version" instead of "revision".

Reply:
Revision should be okay here?